### ISSUE TITLE
self-closing tags?

### ISSUE THREAD

**Issue by billortell** [hamlphp/HamlPHP#9]
:) nuff said?

I keep getting the proverbial

   < input name="name" type="type" value="value"></ input>

suggestions?

**Comment by sniemela** [comment]
Try to add a slash explicitly after the element definition, for instance:

```
%input{value => 'value'}/
```

There is no way to automatically close known self-closing tags(br, link...) at the moment, but the feature is on the backlog.

**Comment by billortell** [comment]
i'll try that again, but i'm _pretty_ sure that didn't work.

**Comment by complex64** [comment]
I vote to reopen the Issue, because as of this writing (67bd43bbf7) adding a backslash does not produce a self-closing tag.

**Example:**

``` haml
%img{src => $image}/
```

Renders to:

``` html
<img  src="..."></img>
```

Instead of the expected:

``` html
<img src="..." />
```

Unnecessary white-space is another issue.

**Comment by sniemela** [comment]
ok, this is fixed in b81593d3febb2b1001a34c609f8f88d73fbb2958
